pkgs/by-name/ca/cargo-ament-build/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "cargo-ament-build";
+  version = "0.1.9";
+  src = fetchFromGitHub {
+    owner = "Guelakais";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "cargo-ament-build";
```
Do not rely on the repo name matching the program's name